spring-kafka/src/main/java/org/springframework/kafka/annotation/KafkaListenerAnnotationBeanPostProcessor.java

```diff
@@ -204,6 +208,20 @@ public void setBeanFactory(BeanFactory beanFactory) {
 		}
 	}
 
+	/**
+	 * Set a charset to use when converting byte[] to String in method arguments.
+	 * Default UTF-8.
+	 * @param charset the charset.
+	 * @since 2.2
+	 */
+	public void setCharset(Charset charset) {
+		Assert.notNull(charset, "'charset' cannot be null");
+		this.charset = charset;
+	}
+
+	private Charset getCharset() {
```

Comment:
What is the purpose of this `private` method?
I don't say that it is harmful somehow, but it is not useful as well...

Reply:
Artifact of an earlier iteration - will remove.